src/main/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/InitAuthenticationRequest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.internship.frejaeidjmeterplugin.jmeter.frejaRequests;
+
+import com.verisec.frejaeid.client.beans.authentication.get.AuthenticationResult;
+import com.verisec.frejaeid.client.beans.authentication.get.AuthenticationResultRequest;
+import com.verisec.frejaeid.client.beans.authentication.init.InitiateAuthenticationRequest;
+import com.verisec.frejaeid.client.beans.general.SslSettings;
+import com.verisec.frejaeid.client.client.api.AuthenticationClientApi;
+import com.verisec.frejaeid.client.client.impl.AuthenticationClient;
+import com.verisec.frejaeid.client.enums.FrejaEnvironment;
+import com.verisec.frejaeid.client.enums.MinRegistrationLevel;
+import com.verisec.frejaeid.client.enums.TransactionContext;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidException;
+
+/**
+ *
+ * @author Ivan
+ */
+public class InitAuthenticationRequest {
+
+    public static AuthenticationResult create(String email, String keyStorePath, String keyStorePassword, String relyingParty, MinRegistrationLevel registrationLevel, String serviceAdress) throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
```

Comment:
My suggestion is to have to separate methods for initiating request and for fetching results.